bb/master/config.py

```diff
@@ -208,22 +208,22 @@ class Mode(Enum):
 GITHUB_OWNER = "Intel-Media-SDK"
 
 CURRENT_MODE = Mode.PRODUCTION_MODE
-#CURRENT_MODE = Mode.PRODUCTION_MODE_PRIVATE
+#CURRENT_MODE = Mode.PRODUCTION_MODE_NEXT_GEN
 #CURRENT_MODE = Mode.TEST_MODE
 
 if CURRENT_MODE == Mode.PRODUCTION_MODE:
     GITHUB_OWNERS_REPO = "MediaSDK"
     BUILDBOT_URL = "http://mediasdk.intel.com/buildbot/"
 
-elif CURRENT_MODE == Mode.PRODUCTION_MODE_PRIVATE:
-    BUILDBOT_TITLE = "MediaSDK Private"
+elif CURRENT_MODE == Mode.PRODUCTION_MODE_NEXT_GEN:
+    BUILDBOT_TITLE = "MediaSDK Next-Gen"
     WORKERS = {"build": {"b-50-41": {},
                          "b-50-61": {}},
                "ubuntu": {"b-999-999": {}},
                "centos_test": {"t-999-999": {}}}
     GITHUB_OWNERS_REPO = msdk_secrets.EMBEDDED_REPO
     BUILDBOT_URL = msdk_secrets.BUILDBOT_URL
-    MASTER_PRODUCT_TYPE = "embedded_private"
+    MASTER_PRODUCT_TYPE = "next_gen"
```

Comment:
If you change this variable need to change product-type in `build_scripts/build_runner.py`